lib/kb_uploadmethods/Utils/ImportSRAUtil.py

```diff
@@ -57,12 +57,26 @@ def _sra_to_fastq(self, scratch_sra_file_path, params):
         tmp_dir = os.path.join(self.scratch, str(uuid.uuid4()))
         handler_utils._mkdir_p(tmp_dir)
 
+        # PTV-1356 fix:  to try to make handing of output files from fastq-dump
+        # more uniform for cases where there is an SRA extension or not
+
+        # force an .sra extension if there isn't one already:
+
+        if not re.match( ".*\.sra$", scratch_sra_file_path, re.IGNORECASE ):
+            new_scratch_sra_file_path = scratch_sra_file_path + ".sra"
+            os.symlink( scratch_sra_file_path, new_scratch_sra_file_path );
```

Comment:
no sure why creating a symbolic link instead of renaming. potential hidden bugs.